features/themes/apply.css

```diff
@@ -925,6 +925,10 @@ a.label:not([style*="color:"]):focus {
 	background-color: var(--input-background-color) !important;
 }
 
+.article-content table tr:nth-child(2n) {
+	background-color: var(--input-background-color) !important;
```

Comment:
For odd table rows, use `--list-odd-background-color`. Additionally, it should probably be `:nth-child(2)`, without the `n`.

Reply:
with `--list-odd-background-color` I have this result : 
<img width="382" alt="image" src="https://user-images.githubusercontent.com/11821952/178553806-5539104f-fe9b-45ab-8b88-9908d40956a3.png">
alternatively I can use `--list-hover-background-color` : 
<img width="364" alt="image" src="https://user-images.githubusercontent.com/11821952/178554001-77d7f9ef-8726-48e1-9673-c3378c8ddaf1.png">

If I don't put the `n` I don't get all the table changed

Reply:
I see. Then this looks fine, merged!